Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -663,6 +663,21 @@ MuteCameraOnMicrophoneInterruptionEnabled:
     WebCore:
       default: false
 
+NativeFiltersEnabled:
```

Comment:
Not really a fan of "Native" - it doesn't really say anything. Maybe "GraphicsContext Filters" or something?

Reply:
Done. This was changed to `GraphicsContextFiltersEnabled`.